lib/topological_inventory/orchestrator/api.rb

```diff
@@ -123,9 +123,11 @@ def tenant_header(tenant_account)
 
       # Set of ids for supported applications
       def supported_application_type_ids
+        return @supported_application_type_ids if @supported_application_type_ids.present?
```

Comment:
If we cache this and never reload it, we wouldn't ever pick up changes to the supported applications until the orchestrator was restarted right?

Maybe we should cache_with_timeout this?

Reply:
@agrare is there some trick how to rewrite object method to cache_with_timeout's class method?
Because it's using other non-class methods and it seems that cache_with_timeout doesn't receive any arguments.

Reply:
We'd have to rewrite this to be a class method (and anything it depends on) which I think is okay because none of this is tenant-level it is all service-level.

Reply:
Arg I just noticed that the sources_api url is an instance var, we could either make this a class var or create our own "instance method" version of cache_with_timeout

Reply:
resolved by class var